libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -151,25 +155,48 @@ class NavigationRoute internal constructor(
             routerOrigin: RouterOrigin,
             routeParser: SDKRouteParser = NativeRouteParserWrapper
         ): List<NavigationRoute> {
+            logI("NavigationRoute.createAsync is called", LOG_CATEGORY)
             return coroutineScope {
                 val deferredResponseParsing = async(ThreadController.DefaultDispatcher) {
-                    DirectionsResponse.fromJson(directionsResponseJson)
+                    DirectionsResponse.fromJson(directionsResponseJson).also {
+                        logI(
```

Comment:
These should rather be `debug` logs, not `info`. What do you think @VysotskiVadim? Could you run with a quick follow-up?

Reply:
I look at debug logs as step-by-step or very granular pieces of information needed to understand the inner status of a feature or unit. (e.g. "download chunk buffer filled, saving to disk and creating new one")

Info logs are pieces that describe the result of multiple operations that is relevant to or all most of developers interacting with a feature. (e.g. "downloaded road shield bitmap")

Warning logs are as info but represents statuses that might require developer action. (e.g. "downloading deprecated shield bitmap, consider changing your config")

Errors logs represent problems that impact the operation of a feature. (e.g. "failed to download shield bitmap)